allred.py

```diff
@@ -0,0 +1,34 @@
+
+import os
+import torch
+from torch._dynamo.utils import same
+from torch._inductor.compile_fx import compile_fx as inductor_compile_fx
+from torch.fx.experimental.proxy_tensor import make_fx
+from torch.fx.passes.graph_manipulation import replace_target_nodes_with
+
+def matmul_cat_col(a, b, c, d, e, f):
+    x = torch.matmul(a, b)
+    y = torch.matmul(c, d)
+    z = torch.cat((x, y))
+    z = torch.ops.c10d.traceable_allreduce(z)
+    g = torch.matmul(e, f)
+    return (torch.add(z, g.repeat(2, 1)), )
+
+def compile(func, example_inputs):
+    graph = make_fx(func)(*example_inputs)
+    replace_target_nodes_with(graph, "call_function", torch.ops.c10d.allreduce_.default, "call_function", torch.ops.c10d.traceable_allreduce.default)
```

Comment:
Given that you're directly using traceable_allreduce, this is not needed , correct?

Reply:
yea, this is old. it's not doing anything. i think i removed already it but i will if not